README.md

```diff
@@ -1,84 +1,86 @@
+Testing windows is not different from regular testing of k8s clusters.  If you have never used a tool such as *e2e.test* or *sonobuoy* to run the Conformance (or other ginkgo based) test suites, then you should familiarize your self with that, first.
 
-Make sure you have a working [Kubernetes development environment](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md) on a Mac or Linux machine. If you're using Windows, you can use WSL, but it will be slower than a Linux VM. The tests can be run from the same VM, as long as you have a working KUBECONFIG.
-
-```bash
-go get -d k8s.io/kubernetes
-cd $GOPATH/src/k8s.io/kubernetes
-./build/run.sh make WHAT=test/e2e/e2e.test
-```
-Once complete, the binary will be available at: `~/go/src/k8s.io/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test`
-
-#### Cross-building for Mac or Windows
+Some resources introducing the e2e test framework:
 
-To build a binary to run on Mac or Windows, you can add `KUBE_BUILD_PLATFORMS`.
+- https://kubernetes.io/blog/2019/03/22/kubernetes-end-to-end-testing-for-everyone/
+- https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#building-kubernetes-and-running-the-tests
 
-For Windows
-```bash
-./build/run.sh make KUBE_BUILD_PLATFORMS=windows/amd64 WHAT=test/e2e/e2e.test
-```
+Now, assuming you are able to build an e2e.test binary, we'll proceed with how windows the tests work.
 
-For Mac
-```bash
-./build/run.sh make KUBE_BUILD_PLATFORMS=darwin/amd64 WHAT=test/e2e/e2e.test
-```
+**See the Questions section at the bottom of this doc** for pointers on how to build the e2e tests. 
 
-Your binaries will be available at `~/go/src/k8s.io/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test` where `linux/amd64/` is replaced by `KUBE_BUILD_PLATFORMS` if you are building on Mac or Windows.
+## Running the e2e.test binary on a windows enabled cluster
 
+If you already have a cluster, you will likely just need to build e2e.test, and run it with windows options.
 
-## Running an e2e test pass
+### Method 1: Using an existing cluster
 
+- FIRST you need to taint/label all linux and master nodes with `node-role.kubernetes.io/master=NoSchedule`... this way, any `windows` specific tests are gauranteed to run on Windows nodes.
 
-### Using an existing cluster
+All of the tests are built into the `e2e.test` binary, which you can as a standalone binary to test an existing cluster. 
+If you don't know how to build `e2e.test`, instructions for this are at the bottom of this doc.
 
-All of the tests are built into the `e2e.test` binary, which you can as a standalone binary to test an existing cluster.
+Note that, when running these **You need to set the KUBE_TEST_REPO_LIST** environment variable when running the windows end to end tests, otherwise, windows images will not be pullable and all your tests will fail !
```

Comment:
Is this really needed?

Reply:
Right now it is required since the windows test images do not live in gcr repository.  There is work in progress for this and should be lit up in 1.21. It will still be required for tests running against 1.20 and earlier as only the new 1.21 tests images will be promoted I believe.